src/services/ai/computer_use/sampling.ts

```
@@ -1,353 +1,428 @@
       for (const action of toolUse) {
-        console.log('Processing tool action:>>', action)
+        console.log("Processing tool action:>>", action);
         // Ensure coordinate format is consistent
         if (action.coordinate && Array.isArray(action.coordinate)) {
-          action.coordinates = { x: action.coordinate[0], y: action.coordinate[1] }
-          delete action.coordinate
+          action.coordinates = {
+            x: action.coordinate[0],
+            y: action.coordinate[1],
+          };
+          delete action.coordinate;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)